sdk/keyvault/keyvault-admin/src/lro/backup/operation.ts

```diff
@@ -58,9 +58,9 @@ export class BackupPollOperation extends KeyVaultAdminPollOperation<
   private async fullBackup(
     options: KeyVaultClientFullBackupOptionalParams
   ): Promise<KeyVaultClientFullBackupResponse> {
-    const span = createSpan("generatedClient.fullBackup", options);
+    const { span, updatedOptions } = createSpan("generatedClient.fullBackup", options);
```

Comment:
Not related to your changes but why is the span `generatedClient.fullBackup` and not `client.fullBackup`? Just curious since this looks like it's in the convenience layer (same applies to other instances of this below)

Reply:
I kicked off another build of KV Admin. The Managed HSM is still in preview but more recent builds have deployed it successfully so let's see how that goes...

Reply:
Ah, I'd talked with @sadasant  about this. I think you'll want to discuss what you want these spans to be called - I think some of this just fell out from the original tracing work but now's a good time to take a step back and make your naming how you think it should be.